packages/core/src/indexing/local-index-api.ts

```diff
@@ -53,6 +54,7 @@ export class LocalIndexApi implements IndexApi {
     networkName: Networks
   ) {
     this.databaseIndexApi = makeIndexApi(indexingConfig, networkName, logger)
+    this.enabled = !indexingConfig?.disableComposedb
```

Comment:
this isn't quite right.  If `indexingConfig` is null this will set `enabled` to true.  The `makeIndexApi` function already has logic to interpret the indexingConfig.  I'd just set `enabled` based on whether `databaseIndexApi` is defined or not.

Reply:
Thanks, should be fixed now